src/features/wallet/Wallet.tsx

```diff
@@ -291,6 +293,11 @@ export const Wallet: FC<WalletPropsType> = ({
           icon="menu"
           iconSize={1.5625}
         />
+        <ChainButton
+          chainSelectionOpen={chainsOpen}
```

Comment:
```suggestion
        <ChainButton
          chainId={chainId}
          chainSelectionOpen={chainsOpen}
```

Use chainId from `const { chainId, account, activate, deactivate, active, library, error } =
    useWeb3React<Web3Provider>();` above.